torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -475,6 +477,18 @@ struct NCCLTraceBuffer {
     return id_++;
   }
 
+  void record_pg(size_t pg_id, std::vector<uint64_t> ranks) {
```

Comment:
nit: more specific, record_pg_ranks?

Reply:
So far I think we only need the ranks involved in a PG so we can use record_pg_ranks for now. If more PG info is needed in the future, we can extend it and rethink the new name.

Reply:
Does it make sense to include the nccl comm id in this too?

Reply:
I think we can revisit this (adding nccl comm id) if the current info is not enough?